### DISCUSSION TITLE
[日服]肉鸽的助戰&開局幹員

### DISCUSSION THREAD

**Discussion by bozoda947** [MaaAssistantArknights/MaaAssistantArknights#3398] [Q&A]
現在選擇不了助戰 , 開局幹員打什麼名字都好像不行....
另外想說 如果不期而遇 出現 以血還血 事件 / 雕刻石像 關卡 的話會卡住
有沒有一個不動一定時間自動退出重來

**Comment by zyxwro** [comment]
美服有同样的问题 一模一样 求大神有空帮忙修修呗
